altair/jupyter/__init__.py

```diff
@@ -0,0 +1,17 @@
+try:
+    import anywidget  # noqa: F401
+    from .jupyter_chart import JupyterChart
+except ImportError:
+    # When anywidget isn't available, create stand-in JupyterChart class
+    # that raises an informative import error on construction. This
+    # way we can make JupyterChart available in the altair namespace
+    # when anywidget is not installed
+    class JupyterChart:  # type: ignore
+        def __init__(self, *args, **kwargs):
+            raise ImportError(
+                "The Altair JupyterChart requires the anywidget \n"
+                "Python package which may be installed using pip with\n"
+                "    pip install anywidget\n"
+                "or using conda with\n"
+                "    conda install -c conda-forge anywidget"
```

Comment:
```suggestion
                "    conda install -c conda-forge anywidget"
                "Afterwards, you will need to restart your Python kernel."
            )
```
How about adding this note? Just because for some other optional dependencies in other packages I can then usually just reexecute that code and it tries to import the package again. I see why we don't do this here but just to avoid confusion.

Reply:
Done in 9df3d525e5d052e82e032c9c48c4cff5139c6eb2